packages/react-components/react-theme-sass/sass/borderRadiusTokens.scss

```diff
@@ -0,0 +1,6 @@
+$borderRadiusNone: var(--borderRadiusNone);
```

Comment:
The variable files are currently produced manually. Once this PR is merged, as a follow up work I plan to generate the variables using the token pipeline (those for the tokens produced by the pipeline).